### DISCUSSION TITLE
Push protection of secret scanning does not work when PR merge

### DISCUSSION THREAD

**Discussion by tadelesh** [community/community#37424] [Other Feature Feedback, Questions, & Ideas]
When I merge a pull request, it seems push protection of secret scanning does not work. The code can be merged successfully, but I receive secret scanning alerts after the merge. I'm wondering whether it is by design?

**Comment by ossanna16** [comment]
Hi there @tadelesh and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
